profiles/common/modules/features/news/news_og/news_og.module

```diff
@@ -28,13 +28,9 @@ function news_og_menu() {
 
 
 /**
- * Implements hook_form_alter().
+ * Implements hook_form_FORM_ID_alter().
  */
-function news_og_form_alter(&$form, $form_state, $form_id) {
-  switch ($form_id) {
-    case 'news_node_form':
-      $desc = $form['field_top_news']['und']['#description'];
-      $form['field_top_news']['und']['#description'] = $desc . " " . t('(The visibility of the news must be set to public to appear on the site homepage)');
-      break;
-  }
+function news_og_form_news_node_form_alter(&$form, $form_state, $form_id) {
+  $desc = $form['field_top_news']['und']['#description'];
+  $form['field_top_news']['und']['#description'] = $desc . " " . t('(The visibility of the news must be set to public to appear on the site homepage)');
```

Comment:
There is a small bug here: the `$form_state` should be by reference. This was already there before, and currently this `$form_state` variable is unused so it doesn't affect the code. It's still a good idea to fix it, otherwise when people work on this code in the future and change the form state it will be very confusing why it doesn't work.

You can also remove the `$form_id` when converting a `hook_form_alter()` to a `hook_form_FORM_ID_alter()`. It's no longer needed then, even though it is still passed to the function.